clicommand/pipeline_upload.go

```diff
@@ -97,6 +98,11 @@ var PipelineUploadCommand = cli.Command{
 			Usage:  "Skip variable interpolation the pipeline when uploaded",
 			EnvVar: "BUILDKITE_PIPELINE_NO_INTERPOLATION",
 		},
+		cli.BoolFlag{
+			Name:   "reject-secrets",
+			Usage:  "When true, fail the pipeline upload early if the the pipeline contains secrets.",
```

Comment:
Style nit: generally our multi-sentence usages have full stops, single sentences don't.

```suggestion
			Usage:  "When true, fail the pipeline upload early if the the pipeline contains secrets",
```